Makefile

```diff
@@ -272,6 +272,13 @@ install-govmomi:
 
 test: install-govmomi portlayerapi $(TEST_JOBS)
 
+local-ci-test:
+ifndef TARGET_VCH
+	@echo To run ci tests, you must set TARGET_VCH to an already deployed VCH
+else
+	@echo running ci-tests locally...
+	infra/scripts/local-ci.sh ./test.secrets.nested
```

Comment:
Use `$(BASE_DIR)/infra/scripts/` as it allows calling from directories other than the build